framework/src/setroubleshoot/browser.py

```diff
@@ -790,12 +790,19 @@ def show_current_alert(self):
         alert.update_derived_template_substitutions()
 
         self.toggles=[]
-        for p, args in plugins:
-               rb = self.add_row(p, alert, args)
 
         if len(plugins) == 1:
+               rb = self.add_row(p, alert, args)
```

Comment:
Traceback (most recent call last):                                                                                                                                                        
       File "/usr/lib/python3.6/site-packages/setroubleshoot/browser.py", line 828, in on_previous_button_clicked                                                                              
         self.show_current_alert()                                                                                                                                                             
       File "/usr/lib/python3.6/site-packages/setroubleshoot/browser.py", line 795, in show_current_alert                                                                                      
         rb = self.add_row(p, alert, args)                                                                                                                                                     
     UnboundLocalError: local variable 'p' referenced before assignment